plugins/ahmadfatoni/apigenerator/controllers/api/UsersAPIController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AhmadFatoni\ApiGenerator\Controllers\API;
+
+use Cms\Classes\Controller;
+
+use Illuminate\Http\Request;
+use AhmadFatoni\ApiGenerator\Helpers\Helpers;
+use Tymon\JWTAuth\Facades\JWTAuth;
+use RainLab\User\Models\User;
+
+class UsersAPIController extends Controller
+{
+    protected $User;
+
+    protected $helpers;
+
+    public function __construct(User $User, Helpers $helpers)
+    {
+        parent::__construct();
+        $this->User    = $User;
+        $this->helpers          = $helpers;
+    }
+
+
+    public function index()
+    {
+        $data = $this->User->select('id', 'name', 'email')->get()->toArray();
+        return $this->helpers->apiArrayResponseBuilder(200, 'success', $data);
+    }
+
+    public function me()
+    {
+        /** @var \Illuminate\Auth\AuthManager */
+        $token = JWTAuth::getToken();
+        $decodedToken = JWTAuth::getPayload($token)->toArray();
+        $id = $decodedToken['sub'];
+
+        $user = $this->User->select('id', 'name', 'email', 'username')->where('id', '=', $id)->first();
```

Comment:
Thêm dấu bằng cũng được, nhưng mặc định nó hiểu so sánh bằng rồi ông nhé
```suggestion
        $user = $this->User->select('id', 'name', 'email', 'username')->where('id', $id)->first();
```

Reply:
Ok ô ơi :D